test/activetasks.js

```diff
@@ -0,0 +1,230 @@
+      console.log = oldConsole;
+      done();
+    });
+
+    lab.test('no arguments', done => {
+      cli().catch(err => {
+        assert.ok(/Usage/.test(err.message));
+        done();
+      });
+    });
+
+    lab.test('does json', done => {
+      nock('http://127.0.0.11')
+        .get('/_active_tasks')
+        .reply(200, []);
+
+      console.log = msg => {
+        assert.deepEqual(msg, []);
+        done();
+      };
+
+      nmo
+        .load({nmoconf: __dirname + '/fixtures/randomini', json: true})
+        .then(() => {
+          cli('http://127.0.0.11');
+        });
+
+    });
+
+    lab.test('rejests errors', done => {
+      cli('http://127.0.0.11')
+      .catch(err => {
+        assert.ok(err);
+        done();
+      });
+    });
+
+    lab.test('Not existing cluster', (done) => {
+
+      cli('wrong').catch(err => {
+        assert.ok(/'wrong' is not a cluster /.test(err.message));
+        done();
+      });
+    });
+
+    lab.test('returns no active tasks for filter', done => {
+      console.log = function (msg) {
+        assert.ok(/for that filter/.test(msg));
+        done();
+      };
+
+      nock('http://127.0.0.11')
+        .get('/_active_tasks')
+        .reply(200, []);
+
+      cli('http://127.0.0.11', 'filter');
+    });
+
+
+    lab.test('returns no active tasks', done => {
+      console.log = function (msg) {
+        assert.ok(/There are no active tasks/.test(msg));
+        done();
+      };
+
+      nock('http://127.0.0.11')
+        .get('/_active_tasks')
+        .reply(200, []);
+
+      cli('http://127.0.0.11');
+    });
+
+    lab.test('returns active tasks', done => {
+      const resp = `[{"node":"node1@127.0.0.1","pid":"<0.8331.5>",
+                    "changes_pending":null,"checkpoint_interval":5000,
+                    "checkpointed_source_seq":"1234",
+                    "continuous":true,"database":null,
+                    "doc_id":null,"doc_write_failures":0,"docs_read":44,
+                    "docs_written":44,"missing_revisions_found":44,
+                    "replication_id":"123123123",
+                    "revisions_checked":44,
+                    "source":"http://127.0.0.1:15984/mango-query/",
+                    "source_seq":"1231",
+                    "started_on":1444135320,
+                    "target":"http://127.0.0.1:15984/test-mango/",
+                    "through_seq":"12313","type":"replication",
+                    "updated_on":1444135325,"user":null}]`;
+
+      nock('http://127.0.0.11')
+        .get('/_active_tasks')
+        .reply(200, resp);
+
+      console.log = msg => {
+        if (!/Active/.test(msg)) { return; }
+
+        assert.ok(/Active Tasks:/.test(msg));
+        done();
+      };
+
+      cli('http://127.0.0.11');
+    });
+
+  });
+
+  lab.experiment('getUrlFromCluster', () => {
+
+    lab.test('it returns url for url passed to it', done => {
+      const url = 'http://127.0.0.1';
+      const out = getUrlFromCluster(url);
```

Comment:
maybe we can put this method even into `utils.js` -  i remember that we used something similiar somewhere else